src/model/channel/sticker.rs

```diff
@@ -0,0 +1,57 @@
+use crate::model::id::{StickerId, StickerPackId};
+
+/// A sticker sent with a message
```

Comment:
```suggestion
/// A sticker sent with a message.
```
Just to add the fullstop.